.github/workflows/master_tests.yml

```diff
@@ -35,10 +41,17 @@ jobs:
             run: |
               conda info
               conda list
-          - name: Install CBC  
+          - name: Install CBC Conda
+            if : matrix.os != 'windows-latest'
             run: |
-              conda install coincbc==2.10.5
-
+              conda install coincbc
+          - name: Install CBC Windows
```

Comment:
The only thing I'd consider changing is possibly combining the two CBC steps into one and using the shell to decide between the two set of commands.  I think this will work:
```run: |
  if [ ${{ matrix.os }} = windows-latest ]
  then
    # download CBC binary, copy to place already in PATH
    curl -O https://ampl.com/dl/open/cbc/cbc-win64.zip
    unzip cbc-win64.zip
    cp cbc.exe $CONDA_PREFIX
  else
    conda install coincbc
  fi
```
The only benefit is that we don't end up with an unused test step in the output.  It's just a preference thing though.

Reply:
So it seems like GitHub actions does not support an if/else construct, which I agree is somewhat annoying.
https://stackoverflow.com/questions/60916931/github-action-does-the-if-have-an-else

Reply:
And that's why you'd put it into the shell script.  The code above is bash script that would all go inside the step's run element.  There would be no if: element in the yaml.

Reply:
Ah okay -- I think I prefer this slightly more explicit approach then, just so the logic is all laid out neatly in the yaml file when it next needs to be updated.